virttest/utils_test/libvirt.py

```diff
@@ -3461,75 +3460,6 @@ def virsh_cmd_has_option(cmd, option, raise_skip=True):
         return found
 
 
-def create_secret(params, remote_args=None):
```

Comment:
@nanli1 If this change will cost too much effort, I'd like to suggest taking  @dzhengfy 's proposal - Add deprecated log in libvirt.create_secret(). For details, please check https://github.com/autotest/tp-libvirt/pull/4081#discussion_r823636255

Reply:
Add deprecated log to create_secret func and not move function

Reply:
@dzhengfy 
@Yingshun 
Could you elaborate more details why deprecate this method?